aecore/txs_eqc.erl

```diff
@@ -441,7 +441,7 @@ response_oracle_args(#{accounts := Accounts, height := Height} = S) ->
            frequency([{99, ?LET(Query, elements(maps:get(queries, S)), Query#query.id)},
                       {1, {?Patron, 2, ?Patron}}]),
           [Height, {Sender, Nonce, Oracle},
-           #{oracle_id => aec_idcreate(oracle, Oracle),
```

Comment:
I see fixed in master https://github.com/Quviq/epoch-eqc/blob/0da9bb4e3cb23a5dab683bad2bbacdc7172ea1ba/aecore/txs_eqc.erl#L444